src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -203,6 +203,13 @@ export const ChannelView: React.FC = () => {
     }
   }
 
+  // At mount set the tab from the search params
+  useEffect(() => {
+    const tabIndex = TABS.findIndex((t) => t === searchParams.get('tab'))
+    if (tabIndex === -1) setSearchParams({ 'tab': 'Videos' }, { replace: true })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why disable the deps? If `searchParams` gets updated externally, we can still navigate, right?

Reply:
thought it might enter an infinite loop since it's changed inside the effect but actually no, it works fine

Reply:
nvm, removing deps again because it bugs out when clicking a video, we only need this effect to run on mount and set the searchParams 'tab' to 'Videos' in case there is  no searchParams set or it's an incorrect one so after mount we don't do anything in it